### DISCUSSION TITLE
Intruducing errors and bugs section

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#23] [Show and tell]
wixii cilad ah halkaan soo dhiga oo iska caawiya. sida balanku ahaa marka hore adinka ayaa is caawin doona hadii aad tihiin @dugsiiyeinc/dhaqsopay-extension-developers ama @dugsiiyeinc/dhaqsopay-website-developers xitaa @dugsiiyeinc/dhaqsopay-server-developers kadib ayaa cilada idin la fiirin doonna Insha'Allah!

# The section:

![image](https://github.com/user-attachments/assets/3c53c6a1-0e6d-4cfc-904b-57ed5ce7f654)

**Comment by apdyqafaar** [comment]
ok bro

**Comment by hayle102** [comment]
![image](https://github.com/user-attachments/assets/a30adbf6-84eb-407e-9763-e4335fe3585b)
Asxaabey wax pull request ah waan samayn la ahay maxa qaldan

**Reply by A-M-Ahmed** [reply]
ciladaan waxa keena folders ka yaalo branch kusha qensid iyo kan yalo main wy kaladuwn yhiin  taas keento waa sida ay aniga ilatahy sidoo kle branch wan sofiryy hadii uaqblo xitaa oo merge ladha ho waxaa baxayo isleyhy dhnm folders markii laga reebo kan aad so galisy , 
marka xlaka sidee buu noqonaa 
kawrn haddii an folders ka aad kushqenesid aad kadxtid , merge aans subino from the main to auth-dashboard
kadib adna tii kutaly aad push sodhahdo kadibna pull request sameno lkn mid niyada kuhy in aad kadaxdirto conflect
option 2 waa is leyhy  60% cml in aaan badbdno wlaga ybaaa in aan adegsano **cherry-pick** from your branch to main
 option 3 same as option 1 we just use rebase

**Reply by A-M-Ahmed** [reply]
@dugsiiyeinc/dhaqsopay-contributors   waxkdhah sidee buu xalka noqonaa, mxaa kena na @sharafdin

**Reply by sharafdin** [reply]
ma is xalisay mise cilad jirto akhii ?

**Reply by Nasrah-muse** [reply]
> ![image](https://private-user-images.githubusercontent.com/189034349/389506007-a30adbf6-84eb-407e-9763-e4335fe3585b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI1NDczMDAsIm5iZiI6MTczMjU0NzAwMCwicGF0aCI6Ii8xODkwMzQzNDkvMzg5NTA2MDA3LWEzMGFkYmY2LTg0ZWItNDA3ZS05NzYzLWU0MzM1ZmUzNTg1Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyNVQxNTAzMjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05YTY4YjJjZjhhNDkzNGJkMjFlY2NhNGUwOGI0ZWZkM2VmZGUxMmEwMzdjYTE0M2UwN2JjYmU0ZTFlNDdhYWY2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.wDEaigSIAMlks2ylqA74VYQN4Obgk7HHKBUKYVMaFLo) Asxaabey wax pull request ah waan samayn la ahay maxa qaldan

Eng waxa uu ku leeyahy "there isn't anything to compare"  waxaa keeni kara ban filayaa in branch aad pull request ka dirayso uusan wax isbadal ah ku jirin ama merge la dhahay marhore, marka isbadal iyo commits cusub samee

**Reply by hayle102** [reply]
Asxaabey ilaa hadda waxba iima xalsamin maxaa xal ah

**Reply by hayle102** [reply]
> > ![image](https://private-user-images.githubusercontent.com/189034349/389506007-a30adbf6-84eb-407e-9763-e4335fe3585b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI1NDczMDAsIm5iZiI6MTczMjU0NzAwMCwicGF0aCI6Ii8xODkwMzQzNDkvMzg5NTA2MDA3LWEzMGFkYmY2LTg0ZWItNDA3ZS05NzYzLWU0MzM1ZmUzNTg1Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyNVQxNTAzMjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05YTY4YjJjZjhhNDkzNGJkMjFlY2NhNGUwOGI0ZWZkM2VmZGUxMmEwMzdjYTE0M2UwN2JjYmU0ZTFlNDdhYWY2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.wDEaigSIAMlks2ylqA74VYQN4Obgk7HHKBUKYVMaFLo) Asxaabey wax pull request ah waan samayn la ahay maxa qaldan
> 
> Eng waxa uu ku leeyahy "there isn't anything to compare" waxaa keeni kara ban filayaa in branch aad pull request ka dirayso uusan wax isbadal ah ku jirin ama merge la dhahay marhore, marka isbadal iyo commits cusub samee

waan soo sameeyay is badal kadib waxaan damcay inan sameyo pull request still waxba ma samayn karo

**Reply by shiinedev** [reply]
Bro waxan u malena inadan git clone dhan adan sameen sabab to ah  xogta mainka talo dhan maba heysatid so qaladka dhankas in oo ka jiro ayan u malena

**Reply by hayle102** [reply]
> Bro waxan u malena inadan git clone dhan adan sameen sabab to ah xogta mainka talo dhan maba heysatid so qaladka dhankas in oo ka jiro ayan u malena

bro waa sax wax clone ah ma samaynin sababto ah horaa u sameeyay waxaa samayn iigu yeeshay commits kale oo 180 commit ayaa isoo raacay oon aniga samaynin

**Reply by shiinedev** [reply]
Ha waa wa loo bahan yahay ine ku so racan   waxba dhib ah makenayan ku was marka kaliya adiga waxa laga raba inad markas waxii aad rabto aad kusii darto sidas inad sameyso aya la rabay bro. Waxa muhiim ah inaad update ki u danbeye heysatid markasto oo mainka wax laga badalo wan inaa dhahda **git pull origin main** si aad update ka u danbeye u hesho sido kalene conflect aad uga badbaado

**Reply by hayle102** [reply]
bro midaasi waan sameeyay lkn sharafdin ayaa igu dhahay lama rabo inay ku soo raacaan commits-kaas adiga ma sidaasa ku shaqaysay

**Reply by shiinedev** [reply]
Bro hade sas tahay waxa la rabe inad mar hore clone gareso intan xogtan mainka lagu so darin. hada xalka inad asi weydiiso aye naqone intan aqanay wey dhaftay zxb. Aniga mar hore ayan clone gareye ka dib ayan waxan sameye in aan waxa ku so daro .

**Reply by hayle102** [reply]
Okey bro

**Reply by A-M-Ahmed** [reply]
bro cilada mtgnthy mise waa xalisy @hayle102

**Reply by hayle102** [reply]
Waa taagantahay bro waxayna u ektahay inaan mar kale samayn doono clone

**Reply by A-M-Ahmed** [reply]
sug waa ku laxalinaa

**Reply by A-M-Ahmed** [reply]
majogtaa

**Reply by A-M-Ahmed** [reply]
discordka adiga miyaa qofka soodhigy

**Reply by hayle102** [reply]
Eng waa  xaliyay branch extension ayaan clone ku sameyay thank you eng

**Reply by A-M-Ahmed** [reply]
ok bro
